features/users/profile.feature

```diff
@@ -53,3 +53,15 @@ Feature: As a user of the site
       | hello world   |
       | hello jupiter |
 
+  Scenario: Having karma count on users profile page
+    Given user "Bob" have karma:
+      | total | hangouts_attended_with_more_than_one_participant |
+      | 20    | 20                                               |
+    When I click on the avatar for "Bob"
+    Then I should be on the "user profile" page for "Bob"
+    And Profile page should have html element with "span.karma-summary" css selector and it should contain "20"
+  
+  Scenario: Having karma count as zero on users profile page with zero hangouts_attended_with_more_than_one_participant
+    When I click on the avatar for "Alice"
+    Then I should be on the "user profile" page for "Alice"
+    And Profile page should have html element with "span.karma-summary" css selector and it should contain "0"
```

Comment:
this is great @praveendhawan , but the step could be shorter, perhaps just 

```
And the karma summary is 0
```
mentioning the html element and css selector in the step is very transparent, but ideally cucumber scenarios are at the level of the user experience rather than specifying the implementation